pkgs/development/libraries/openldap/default.nix

```diff
@@ -8,6 +8,14 @@ stdenv.mkDerivation rec {
     sha256 = "091qvwk5dkcpp17ziabcnh3rg3m7qwzw2pihfcd1d5fdxgywzmnd";
   };
 
+  patches = [
+    (fetchurl {
+      url = "https://bz-attachments.freebsd.org/attachment.cgi?id=183223";
```

Comment:
Is this url stable?

Reply:
I... think so? It's linked from the bug tracker at:
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=219781

Clicking around, I find a link to it:
https://svnweb.freebsd.org/ports/head/net/openldap24-server/files/patch-libressl?revision=442605&view=co&pathrev=442605

via:
https://svnweb.freebsd.org/ports/head/net/openldap24-server/files/patch-libressl?view=markup&pathrev=442605

Would this be more appropriate? I can change it if you think so.

Reply:
svn is better, use also fetchpatch instead of fetchurl for further normalization.